recipe/gen_patch_json.py

```diff
@@ -1187,12 +1187,16 @@ def _gen_new_index(repodata, subdir):
         if record_name == "google-api-core" and record["version"] == "1.31.2":
             deps = record["depends"]
             _replace_pin("google-auth >=1.25.1,<3.0dev", "google-auth >=1.25.1,<2.0dev", deps, record)
-        
+
         # auto-sklear needs to depend on the full dask.
         # https://github.com/automl/auto-sklearn/issues/1256
         if record_name == "auto-sklearn":
             _rename_dependency(fn, record, "dask-core", "dask")
 
+        # remove libcugraph constrain to prevent cyclical dependency issues with RAPIDS
+        if record_name == "cupy":
+            _replace_pin("libcugraph >=0.19.0,<1.0a0", "libcugraph", record.get("constrains", []), record, target='constrains')
```

Comment:
As discussed internally:
```suggestion
            _replace_pin("libcugraph >=0.19.0,<1.0a0", "libcugraph >=0.19.0", record.get("constrains", []), record, target='constrains')
```